site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5267,11 +5267,11 @@ public function insertNotifications(array $flattened_notifications, int $notific
      */
     public function insertEmails(array $flattened_params, int $email_count) {
         // PDO Placeholders
-        $row_string = "(?, ?, current_timestamp, ?, ?, ?, ?)";
+        $row_string = "(?, ?, current_timestamp, ?, ?, ?, ?, ?)";
```

Comment:
(leaving a comment here since I can't comment on lines you didn't change)

Can you add a PHPDoc comment specifying the [array shape](https://phpstan.org/writing-php-code/phpdoc-types#array-shapes) for `$flattened_params`?  As it is now, it's difficult to understand what this array contains if you were to only see a small portion of the relevant code.

Reply:
Done!